CHANGES.md

```diff
@@ -4,6 +4,8 @@ Full changelog
 v1.4.0 (2022-05-31)
 -------------------
 
+* Fixed a bug where minor tick marks were not respecting the settings colors. [#2305]
```

Comment:
Can you move this to 1.6.0? (I am going to work on a way to avoid us having to do this in future)